database.go

```diff
@@ -85,7 +85,8 @@ func createDatabase() {
 		last_name TEXT NOT NULL,
 		email TEXT NOT NULL UNIQUE,
 		username TEXT NOT NULL UNIQUE,
-		password TEXT NOT NULL
+		password TEXT NOT NULL,
+		user_type TEXT NOT NULL
```

Comment:
Another possibility is not using a string but a boolean `is_root`. It is less powerful if we want to have different kinds of users in the future.

Reply:
I implement this option instead of the previous one